src/runtime/graph_executor/debug/graph_executor_debug.cc

```diff
@@ -362,6 +396,33 @@ PackedFunc GraphExecutorDebug::GetFunction(const std::string& name,
       ICHECK_GE(min_repeat_ms, 0);
       *rv = this->RunIndividual(number, repeat, min_repeat_ms);
     });
+  } else if (name == "run_individual_node") {
+    return PackedFunc([sptr_to_self, this](TVMArgs args, TVMRetValue* rv) {
```

Comment:
FYI, you can use `TypedPackedFunc` with a lambda to avoid manually unpacking the args.

Reply:
Thanks, done